src/logic/notifications.ts

```diff
@@ -110,6 +110,5 @@ export async function sendNotification(
       senderAddress: context.senderAddress,
       receiverAddress: context.receiverAddress
     })
-    throw error
```

Comment:
What would happen now if we fail to send a notification? will it be retried later or we will show an error message on client side? how is that handle?